server/routes/api/subscriptions/subscriptions.ts

```diff
@@ -143,11 +144,11 @@ router.get(
 
     authorize(user, "delete", subscription);
 
-    ctx.context = {
-      auth: { user, type: AuthenticationType.API },
-      transaction,
+    ctx.context = createContext({
```

Comment:
You can pass `createContext` straight through to `destroyWithCtx`, no need to override